test/web-platform-tests/to-run.yaml

```diff
@@ -392,7 +388,19 @@ custom-element-reaction-queue.html:
   "Mutating a undefined custom element while upgrading a custom element must not enqueue or invoke reactions on the mutated element": [fail, document.write() implementation is not spec compliant]
 element-internals-aria-element-reflection.html:
   "Getting previously-unset ARIA element reflection properties on ElementInternals should return null.": [fail, Unknown]
-form-associated/**: [fail-slow, Not implemented]
+form-associated/ElementInternals-NotSupportedError.html: [fail-slow, Not implemented]
```

Comment:
It looks like these now fail fast, instead of slow, so I'll switch these to `fail`. (Which causes them to run, and be expected to fail, instead of be skipped. So that in the future if someone makes them pass, we notice.)